setup.py

```diff
@@ -11,7 +11,8 @@
                       'jsonpickle',
                       'six',
                       'basictracer>=2.2,<2.3',
-                      'googleapis-common-protos==1.5.3'],
+                      'googleapis-common-protos==1.5.3',
+                      'requests==2.19.1'],
```

Comment:
Does this have to be a hard equality? Can we be more flexible with the requirement here, given that requests could be used in the enclosing application too?